### PR TITLE
Simple AppInsights and logging configuration

### PR DESCRIPTION
**Pull request by dirkrheeder**
Just adds AppInsights to the project and configured the standard logger to log to app insights.

### CODE REVIEW

File: template/AksApi.csproj [hunk 1]
```diff
@@ -5,6 +5,8 @@
   </PropertyGroup>
   <!--#if (enableOpenApi) -->
```
Comment (robbell) [inline template/AksApi.csproj:6]: Think we might need to change this line for the inclusion of the ItemGroup tag: 
```
<!--#if (enableLogging or enableOpenApi) -->
<ItemGroup>
...
```
Then have separate conditions around the two sets of PackageReferences:
```
<!--#if (enableLogging) -->
<PackageReference ...LoggingPackages...>
...
```

File: template/appsettings.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (robbell) [inline template/appsettings.json:1]: We can modify /template/.template.config/template.json to exclude this file if logging is disabled - something like:
```
...
    "sources": [
        {
            "modifiers": [
                {
                    "condition": "(disable-health-check)",
                    "exclude": [
                        "HealthChecks/**"
                    ]
                },
                {
                    "condition": "(disable-logging)",
                    "exclude": [
                        "appsettings.json"
                    ]
                }
            ]
        }
    ]
```

File: template/AksApi.csproj (hunk 1 above)
Comment (dirkrheeder) [reply template/AksApi.csproj:6]: I've created two item groups, one for each. Tested with a build and it seems happy

File: template/appsettings.json (hunk 2 above)
Comment (dirkrheeder) [reply template/appsettings.json:1]: done and tested

**Review by robbell — APPROVED** [review]

**Pull request merged**